### ISSUE TITLE
aiomysql-0.0.17 hash on pypi is changed

### ISSUE THREAD

**Issue by pyk** [aio-libs/aiomysql#318]
Hi, I got the following error from my CI/CD pipeline:

```
An error occurred while installing aiomysql==0.0.17! Will try again.
Installing initially failed dependencies...
THESE PACKAGES DO NOT MATCH THE HASHES FROM Pipfile.lock!. If you have updated the package versions, please update the hashes. Otherwise, examine the package contents carefully; someone may have tampered with them.
    aiomysql==0.0.17 from https://files.pythonhosted.org/packages/be/6a/6ab59df05a223969ceb8f457e628fd1616a57c4567b6caa061d984849d2b/aiomysql-0.0.17-py3-none-any.whl#sha256=3b6675f43017ce3b4ec411e8430bc95de1d460ac5bbd1e6eae1391b0842bb93d (from -r /tmp/pipenv-mskxzame-requirements/pipenv-5ahjqir1-requirement.txt (line 1)):
        Expected sha256 85a8e00ab9a1ed53b58e90d228c5539775aabcbe338dfc32d75a97ca54aca0e9
             Got        3b6675f43017ce3b4ec411e8430bc95de1d460ac5bbd1e6eae1391b0842bb93d
```

I don't update the `aiomysql` version on my service. So, i want to clarify on the `aiomysql` maintainer make sure the package is not tampered.

**Comment by pyk** [comment]
The content of the `aiomysql-0.0.17` is available here https://github.com/pyk/aiomysql-0.0.17-py3-none-any

**Comment by pyk** [comment]
Currently, i'm upgrading to `0.0.19`. I will close this issue. Feel free to re-open if this is a real issue

**Comment by terrycain** [comment]
Yeah I think I caused that the other day.

**Comment by mal** [comment]
@terrycain is it possible that this broke the `pymysql` dependency fix? When installing `aiomysql==0.0.17` from a `requirements.txt` , I see:

```
Collecting PyMySQL<=0.9.2,>=0.9 (from aiomysql==0.0.17->-r requirements.txt (line 3))
```

And the metadata in the [updated package](https://github.com/pyk/aiomysql-0.0.17-py3-none-any/blob/master/aiomysql-0.0.17.dist-info/METADATA#L23) contents contain:

```
Requires-Dist: PyMySQL (<=0.9.2,>=0.9)
```

Is it possible to fix this so that `0.0.17` uses the correctly pinned version of `pymysql`?

**Comment by and800** [comment]
@mal imo overwriting existing packages is not the best idea. `0.0.17` wheel is broken, let's just use `0.0.19`
